niceml/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "0.6.0"
+__version__ = "0.6.1"
```

Comment:
Why is this part of your pull request?

Reply:
This was accidentaly added, because the feature branch initially started from the main branch. The changes were removed from this PR.